cmd/agent/Dockerfile

```diff
@@ -1,13 +1,18 @@
-FROM alpine:latest as certs
-RUN apk add --update --no-cache ca-certificates
+ARG base_image
+ARG debug_image
 
-FROM scratch
+FROM $base_image AS release
 ARG TARGETARCH=amd64
 ARG USER_UID=10001
```

Comment:
@Ashmita152 was there a reason for TARGETARCH and USER_UID args to be repeated in each target, instead of being defined at the top of the file?

If not, probably another opportunity to clean up / DRY the docker files.

Reply:
Sure, I will clean that today.